examples/wordRelevance/index.js

```diff
@@ -0,0 +1,55 @@
+require( "console.table" );
+
+var Paper = require( "../../js/values/Paper" );
+var relevantWords = require( "../../js/researches/relevantWords" )
+var getRelevantWords = relevantWords.getRelevantWords;
+var getWordCombinations = relevantWords.getWordCombinations;
+var calculateOccurrences = relevantWords.calculateOccurrences;
+var getRelevantCombinations = relevantWords.getRelevantCombinations;
+var WordCombination = require( "../../js/values/WordCombination" );
+var formatNumber = function ( number ) {
+
+   if ( Math.round( number ) === number ) {
+      return Math.round( number );
```

Comment:
Why not return number here. If Math.round( number ) is the same as number, why round it again?